src/main-layout/SideMenu/SidePanelContainer/GlobalParams/GlobalParamsForm.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+    PlusOutlined,
+    DeleteOutlined
+} from '@ant-design/icons';
+import { Button, Popover } from 'antd';
+import React from 'react';
+import { LM } from 'src/translations/language-manager';
+import "../../../../main-layout/SessionWindow/SessionTab/SessionManagement/ParamsForm.scss";
+import { GlobalParameterService } from "../../../../services/GlobalParameterService";
+import { GlobalServiceRegistry } from "../../../../services/GlobalServiceRegistry";
+import { ParameterForm } from "../../../SessionWindow/SessionTab/SessionManagement/SessionParamsForm";
```

Comment:
Better if we can rename the SessionParamsForm -> ParamsForm maybe

Reply:
Refactored ParameterForm into a standalone file